docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -0,0 +1,245 @@
+# Quickstart
+
+<p class="description">Get started with Base UI, a library of headless ("unstyled") React UI components and low-level hooks.</p>
+
+## Installation
+
+`@mui/base` is completely standalone – run one of the following commands to add Base UI to your React project:
+
+### With npm
+
+```sh
+npm install @mui/base
+```
+
+### With yarn
+
+```sh
+yarn add @mui/base
+```
+
+### With pnpm
+
+```sh
+pnpm add @mui/base
+```
+
+### Peer dependencies
+
+<!-- #react-peer-version -->
+
+[`react`](https://www.npmjs.com/package/react) >= 17.0.0 and [`react-dom`](https://www.npmjs.com/package/react-dom) >= 17.0.0 are peer dependencies.
+
+## Implementing a Button
+
+This is a quick tutorial that goes through the basics of using and styling Base UI components by replicating a button from GitHub's UI, using their [Primer design system](https://primer.style) as a reference.
+Base UI components help you abstract away low-level UI implementation details such as accessibility patterns, cross-browser compatibility, event handling etc while remaining unopinionated about styling, giving you complete control over your app's CSS.
```

Comment:
> I wonder if it is worth adding a paragraph that explains why you should use Base UI button instead of a plain HTML button.

@samuelsycamore I've added a sentence here, not sure if it's the smoothest transition 😅

Reply:
Should we add a heading to separate these two? Quite directly "Why use Base UI's button instead of native HTML?" or something like that. 😬

Reply:
@danilo-leal I think that's a great idea, but I wonder if this is the best place for it? I would assume that if you've made it to the Quickstart, you've already concluded that you want to use Base's Button because it's better than an HTML button. 😁 Maybe this calls for an update to the Overview page?

Reply:
Avoid Latinisms whenever possible 😇

```suggestion
Base UI components help you abstract away low-level UI implementation details such as accessibility patterns, cross-browser compatibility, event handling, and more, while remaining unopinionated about styling, giving you complete control over your app's CSS.
```

Reply:
Oh, yeah ⎯ that checks out! Might make sense to tackle it together with the "pro vs. con" thread that was going on. Base UI should add some advantage against native HTML 😬